Makefile

```diff
@@ -84,6 +84,9 @@ bin/unpack:
 build-local-container: ## Builds provisioner container image locally
 	docker build -f Dockerfile -t $(IMAGE) .
 
+build-container: build
```

Comment:
It would be nice to hook this target into the kind-load target to help enable better dev workflows:

- build the local dockerfile
- load that local image into kind
- respin the provisioner deployment pod manually?

Maybe this is where the (now removed in #79) `run-local` target could come back into play again, and that target builds the local image and loads it into kind?

Reply:
It's nice.  I'll look at the dev workflow.